torch/csrc/jit/codegen/cuda/kernel_ir_printer.h

```diff
@@ -20,13 +20,17 @@ namespace kir {
 //!
 //! This class is intended for debug printing, so it attempts
 //! to handle invalid IR states as much as possible.
-//!
 class TORCH_CUDA_API IrPrinter : private kir::IrVisitor {
   static constexpr char* kTab = "  ";
 
  public:
   //! Constructs a new IrPrinter which outputs to the specified stream
-  explicit IrPrinter(std::ostream& os) : os_(os) {}
+  //!
+  //! When implicit_definition is true, recurisvely print the
```

Comment:
same question: why/where do we need this new  knob?

Reply:
Not sure actually. @csarofeen ?